src/readers/morphologyHDF5.cpp

```diff
@@ -213,6 +213,35 @@ void MorphologyHDF5::_readPoints(int firstSectionOffset) {
     auto& somaPoints = _properties._somaLevel._points;
     auto& somaDiameters = _properties._somaLevel._diameters;
 
+    using float_2dvec = std::vector<std::vector<float>>;
```

Comment:
I'm not sure the alias is necessary considering you use it only once. Besides, `vec` could be called `hd5fData` or something that suggests this is the input data not the output ones.

Reply:
Even though new I think it was a mistake, MorphIO uses camel-case, so `has_neurites` -> `hasNeurites_`